datadog_cluster_agent/datadog_checks/datadog_cluster_agent/check.py

```diff
@@ -68,6 +68,7 @@
     'kubernetes_apiserver_kube_events': 'kubernetes_apiserver.kube_events',
     'language_detection_dca_handler_processed_requests': 'language_detection_dca_handler.processed_requests',
     'language_detection_patcher_patches': 'language_detection_patcher.patches',
+    'leader_election_is_leader': 'leader_election.is_leader',
```

Comment:
this metric wasn't designed to be exposed directly. 
Instead it was added to be use as "label join", see: https://github.com/DataDog/integrations-core/blob/c2c5bcd5db8e7f5db93109411338a6c8c811bbdd/datadog_cluster_agent/datadog_checks/datadog_cluster_agent/check.py#L100-L105

IMO, if we want to make a metric that exposes the leader status we should use the metric value (`1` if leader, `0` if follower) instead of the using a label. 
Maybe it can be done in this check thanks to a metric transformer function that will convert the `is_leader` label's value to a metric value.

you can find some transformer example in other openmetrics base check like https://github.com/DataDog/integrations-core/blob/642b2f9f30a05c09f32025a9176d0e1a13cf8f61/kube_controller_manager/datadog_checks/kube_controller_manager/kube_controller_manager.py#L231

Reply:
> IMO, if we want to make a metric that exposes the leader status we should use the metric value (1 if leader, 0 if follower) instead of the using a label.

What do you think if I achieve that in Cluster Agent side? https://github.com/DataDog/datadog-agent/pull/32511

Reply:
> Also it would be nice to add the metric in the OOTB Dashboard too.

Will do!